### ISSUE TITLE
[FEAT] Making a COPY and PASTE button

### ISSUE THREAD

**Issue by shravanibhor** [singodiyashubham87/Draw-it-out#128]
If the user has to draw the same things again and again, one can use the copy and paste button such that one can copy the necessary section and paste it on the white board wherever needed.

**Comment by 0xabdulkhalid** [comment]
Hi there @shravanibhor, You need to work on One issue at a time.

Please work on #127 for now, once you completed that successfully i'll assign this one to you!

**Comment by 0xabdulkhalid** [comment]
Hi @shravanibhor, can you please elaborate how you're going to implement the Copy/Paste feature ?

How are you going to do such actions with the Strokes drawn on top of canvas?

Let me ask @singodiyashubham87 (Project Admin) regarding this one!

**Comment by shravanibhor** [comment]
Yes Sure, so there would be two buttons, one to copy and the other one to paste. Once the copy button is clicked the user would get an option to choose the area on the white board which he wishes to copy. The area would then get copied. Further when the paste button is clicked, the user can drag that piece of area wherever he wants to place on the white board.

**Comment by 0xabdulkhalid** [comment]
Seems like a good one, I'm assigning this issue to you @shravanibhor.

Best of luck, make sure to create a PR once you made the right changes!

**Comment by singodiyashubham87** [comment]
@shravanibhor Do remember to follow the standard coding practice, Best of Luck!

**Comment by Tani1126** [comment]
pls, assign this project to me ,I am contributor of GSSOC'24

**Comment by 0xabdulkhalid** [comment]
Sorry @Tani1126, We can't assign this issue for you.

Issues will be only assigned as FCFS (First Come First Serve) Basis, So you need to create a new issue regarding any improvements or bugs. We're here to review it and assign it to you.

Thank you!

**Comment by Tani1126** [comment]
Thankyou sir

**Comment by Mahimatestgithub** [comment]
I want to work in this issue. Please assign me this task.

**Comment by singodiyashubham87** [comment]
@shravanibhor Are you working on the issue?

**Comment by singodiyashubham87** [comment]
@shravanibhor Removing your assignment due to inactivity.

**Comment by singodiyashubham87** [comment]
@Tani1126 Want to work on the issue?

**Comment by Tani1126** [comment]
@singodiyashubham87 @0xabdulkhalid please tell me clearly so I can start my work.

**Comment by singodiyashubham87** [comment]
@Tani1126 You can start working, & sorry for the delay, we are also having our exams, so try to understand :)

**Comment by Tani1126** [comment]
No problem thanks for assigning me this task
